pkg/api/v1alpha1/cluster.go

```diff
@@ -363,6 +364,16 @@ func validateClusterConfigName(clusterConfig *Cluster) error {
 	return nil
 }
 
+func validateUnsupportedFeatures(cluster *Cluster) error {
```

Comment:
Why don't we name this something more specific? Like `validateExternalEtcdSupport`

Reply:
I think that would be good - it will be more descriptive now, and can be modified later if we need to fit other things under it.